### DISCUSSION TITLE
osu lazer crashes after opening

### DISCUSSION THREAD

**Discussion by puppybrained** [ppy/osu#22463] [Q&A]
### Type

Crash to desktop

### Bug description

at some point my osu lazer just started crashing as soon as i open it. it shows the black screen with text that shows before the game loads and then crashes. i dont get any window telling me why or anything at all. i have tried to reinstall it but the issue keeps happening.

### Screenshots or videos

https://user-images.githubusercontent.com/123875527/215327533-687d425a-e5eb-409f-af3e-96b657716653.mp4

### Version

the latest one, obviously i cant really say which exactly since.. it crashes right away

### Logs

[runtime.log](https://github.com/ppy/osu/files/10529800/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10529801/updater.log)
[performance.log](https://github.com/ppy/osu/files/10529802/performance.log)
[network.log](https://github.com/ppy/osu/files/10529803/network.log)
[input.log](https://github.com/ppy/osu/files/10529804/input.log)
[database.log](https://github.com/ppy/osu/files/10529805/database.log)

**Comment by agatemosu** [comment]
Did you try renaming the folder `%appdata%\osu` (so that it creates a new folder)?

**Comment by puppybrained** [comment]
> Did you try renaming the folder `%appdata%\osu`?

wait, ive noticed i have like.. three osu lazer folders.. do u think that could be a problem LMAO

**Comment by puppybrained** [comment]
okay i fixed it, turns out it was because i had two osu lazer folders for some reason, one being basically empty

**Comment by puppybrained** [comment]
i take it back. its not working again and crashing like before

**Comment by bdach** [comment]
Please try updating graphics drivers and disabling shadowplay if enabled.
